snapcraft.yaml

```diff
@@ -29,9 +37,6 @@ apps:
   rasm2:
     command: usr/bin/rasm2
     plugs: [home]
-  sdb:
-    command: usr/bin/sdb
-    plugs: [home]
   rabin2:
```

Comment:
I think it also makes sense to add `rasign2` command, since it was reintroduced recently.

Reply:
I added `rasign2` in the new commit.